index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+    type="text/css" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Home</title>
+</head>
+<body>
+
+ <div class="mob_menu" id="m_menu">
+  <ul>
+    <li class="header-link"><a href="index.html">Home</a></li>
+    <li class="header-link"><a href="about.html">About</a></li>
+    <li class="header-link"><a href="#project_schedule">Program</a></li>
+    <li class="header-link"><a href="./about.html#previous_events">Previous Events</a></li>
+    <li class="header-link"><a href="#art_guests">Guests</a></li>
+  </ul>
+    <img src="./Imgs/Icon.png" alt="Close " class="close" id="close-menu">
+ </div>
+
+  
+  <header id="main-nav">
+    <div class="logout-nav">
+      <nav id="first-nav">
+        <ul>
+          <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <div class="hamburger">
+      <i class="fas fa-bars fa-2x" id="ham_btn"></i>
+    </div>
+    <div class="nav_logo" id="main_nav">
+     <div>
+
+        <a href="#"><img src="./Imgs/logo.png" alt="Logo"></a>
+      </div>
+
+      <nav>
+        <ul>
+          <li class="header-link"><a href="#" class="current">Home</a></li>
+          <li class="header-link"><a href="about.html">About</a></li>
+          <li class="header-link"><a href="#project_schedule">Program</a></li>
+          <li class="header-link"><a href="#">Previous Events</a></li>
+          <li class="header-link"><a href="#art_guests">Guests</a></li>
+        </ul>
+      </nav>
+    </div>
```

Comment:
- [ ] OPTIONAL
The menu items of your mobile menu overlay **cannot be seen clearly**. You can increase the opacity of the mobile menu so that the background beneath your mobile menu overlay becomes blurred, hence making the menu items clearer on the menu overlay.

**Your mobile menu overlay:**
![image](https://user-images.githubusercontent.com/12745474/133712691-39286810-185d-4114-bf1b-b7b23d43a75b.png)

Reply:
- [ ] This issue has not been addressed yet. Please apply the suggestion given to you by the TSE

Reply:
Hello, thank you for the feedback, but the past reviewer said it is optional, I am gonna work on it after I get my project approved